### PR TITLE
[I18N] password_security: Translate to Spanish all translatable terms

### PR DESCRIPTION
**Pull request by luisg123v**
This translates to Spanish all missing translationgs, 28 in total.

### CODE REVIEW

**Comment by luisg123v** [comment]
CC @yaniaular

**Comment by luisg123v** [comment]
@moylop260 
Could you merge this PR, please?

File: password_security/i18n/es.po [hunk 1]
```diff
@@ -55,14 +55,16 @@ msgstr "Fecha"
...
-msgstr ""
+msgstr "Días"
 
 #. module: password_security
 #: model:ir.model.fields,help:password_security.field_res_company_password_history
 msgid ""
 "Disallow reuse of this many previous passwords - use negative number for "
 "infinite, or 0 to disable"
 msgstr ""
+"No permitir este número de contraseñas previas- use números negativos para"
```
Comment (yaniaular) [inline password_security/i18n/es.po]: @luisg123v Review this please

Comment (yaniaular) [reply password_security/i18n/es.po]: never mind

**Comment by yaniaular** [comment]
:+1:

File: password_security/i18n/es.po (hunk 1 above)
Comment (luisg123v) [reply password_security/i18n/es.po]: Done

**Review by yaniaular — APPROVED** [review]
looks good for me

**Pull request merged**